portfolio/src/main/webapp/script.js

```diff
@@ -14,21 +14,32 @@
 
 
 function getGame() {
-    fetch('/data').then(response => response.json()).then((game) => {
-        generateBoard();
+    fetch('/view-games').then(response => response.json()).then((games) => {
         // stats is an object, not a string, so we have to
         // reference its fields to create HTML content
-        const playersContainer = document.getElementById('players-container');
-        playersContainer.innerText = game.white + ' (white) vs ' + game.black + ' (black)'; 
-        const moveListElement = document.getElementById('game-container');
-        moveListElement.innerHTML = '';
-        for(var ii = 0; ii < game.moves.length; ii++){
-            moveListElement.appendChild(
-                createListElement(ii + '. ' + game.moves[ii].first + ', ' + game.moves[ii].second));
+        const allGamesContainer = document.getElementById('games-container');
+
+        for(var ii = 0; ii < games.length; ii++){
+            allGamesContainer.appendChild(createGameElement(games[ii]));
         }
     });
 }
 
+function createGameElement(game){
+    const gameElement = document.createElement('div');
+    gameElement.className = 'game-element'
+    var build = '';
+    build += '<h3>' + game.white + ' (white) vs ' + game.black + ' (black)</h3>';
+    build += '<ol>';
+    for(var ii = 0; ii < game.moves.length; ii++){
+        build += '<li>' + game.moves[ii].first + ', ' + game.moves[ii].second + '</li>';
```

Comment:
I wonder if you can create this somehow by building up a list of <li> objects and appending them to the document rather than by building up a string. That might be a little less prone to errors, but this is ok for now.

Reply:
See how Lee does it here https://github.com/leeannachen/step2020/pull/4/files#diff-6a2fb93ff673d3049df2c1401212e439R53-R57